src/main/java/org/folio/spring/ApplicationConfig.java

```diff
@@ -28,11 +29,19 @@
   "org.folio.validate"})
 public class ApplicationConfig {
 
+  @Bean
+  public PropertySourcesPlaceholderConfigurer placeholderConfigurer() {
+    PropertySourcesPlaceholderConfigurer configurer = new PropertySourcesPlaceholderConfigurer();
+    configurer.setLocation(new ClassPathResource("application.properties"));
```

Comment:
I think it's better to rename "application.properties" to something else, for example custom-field-application.properties.
I think if this configuration runs in some other module, (e.g. mod-kb-ebsco-java) it will pick up application.properties from mod-kb-ebsco-java, instead of from mod-custom-fields

Reply:
@andrii-paias renamed